pkg/runtime/kubernetes/utils.go

```diff
@@ -43,12 +43,12 @@ func GetKubectlAndKubeconfig(ssh ssh.Interface, host net.IP, rootfs string) erro
 		return errors.Wrap(err, "failed to add master IP to etc hosts")
 	}
 
-	if !osi.IsFileExist(common.KubectlPath) {
-		err = osi.RecursionCopy(filepath.Join(rootfs, "bin/kubectl"), common.KubectlPath)
+	if !osi.IsCommandExist("kubelet") {
```

Comment:
Just double check if here is correct.  `kubelet` or `kubectl`? @yummypeng @kakaZhou719

Reply:
@allencloud  my mistake, update :)